newIDE/app/src/Utils/GDevelopServices/Build.js

```diff
@@ -61,6 +61,29 @@ export const getBuildArtifactUrl = (
   return `https://builds.gdevelop-app.com/${build[keyName]}`;
 };
 
+export const getWebBuildGameFolderUrl = (buildId: string): string => {
+  return `https:/games.gdevelop-app.com/game-${buildId}`;
+};
+
+export const getWebBuildThumbnailUrl = (
+  project: gdProject,
+  buildId: string
+): string => {
+  // TODO use some helper methods to do this?
+  const path = project
+    .getPlatformSpecificAssets()
+    .get('liluo', `thumbnail`)
+    .split('\\');
```

Comment:
That looks like it would break if a non-windows path is used?

Reply:
Yes, this is a temporary hack, there is a TODO right before.

Reply:
Ah, right 😅 My brain automatically skips those 😓